fixtures/sponsors.json

```diff
@@ -4,7 +4,6 @@
         "added": "2016-04-11T21:55:00Z",
         "name": "Continuum Analytics",
         "display_url": "",
-        "sponsor_logo": 4,
```

Comment:
I think it would be best if this commit was removed from the pull request, since it doesn't have to do with adding API support for presentations or sponsors.

Since the intent of this fixture is to automatically have Continuum Analytics' logo show up on every conference site, we should fix this bug by automatically adding the logo instead of removing the foreign key relationship.

I just saw [you asked about this a few months ago](https://github.com/pydata/conf_site/pull/117#discussion_r66953141) - sorry that I missed your question then.